README.md

```diff
@@ -57,10 +38,134 @@ The following commands are for Ubuntu Server. However, equivalent commands shoul
+
+```shell
+AWS_ACCESS_KEY_ID
+AWS_SECRET_ACCESS_KEY
+AWS_DEFAULT_REGION
+AWS_ENDPOINT
+AWS_ALLOW_HTTP
+```
+
+For example, to use a local instance of [Minio](https://min.io/), assuming the access key id `KURo1eQeMhDeVsrz` and the secret access key `sp7TDyD2ZruJ0VdFHmkacT1Y90PVPF7p`has been created through [Minio's web interface](http://127.0.0.1:9001/access-keys), set the environment variable as follows:
+
+```shell
+AWS_ACCESS_KEY_ID="KURo1eQeMhDeVsrz"
+AWS_SECRET_ACCESS_KEY="sp7TDyD2ZruJ0VdFHmkacT1Y90PVPF7p"
+AWS_DEFAULT_REGION=""
+AWS_ENDPOINT="http://127.0.0.1:9000"
+AWS_ALLOW_HTTP="true"
+```
+
+Then, assuming a bucket named `wind-turbine` has been created through [Minio's web interface](http://127.0.0.1:9001/buckets), `modelardbd` can be run in edge mode with automatic transfer of the ingested time series to the Minio bucket `wind-turbine`:
+
+```shell
+modelardbd edge path_to_local_data_folder s3://wind-turbine
+```
+
+To run `modelardbd` in cloud mode simply replace `edge` with `cloud` as shown below. Be aware that both a local data folder and an object store are required when `modelardbd` is run in cloud mode.
+
+```shell
+modelardbd cloud path_to_local_data_folder s3://bucket-name
+```
+
+### Execute SQL
+ModelarDB includes a command-line client in the form of `modelardb`. To interactively execute SQL statements against a local instance of `modelardbd` through a REPL, simply run `modelardb`:
+
+```shell
+modelardb
+```
+
+If `modelardbd` is not running on the same host, the host `modelardb` should connect to must be specified:
+
+```shell
+modelardb 127.0.0.1
+```
+
+`modelardb` can also execute SQL statements from a file passed as a command-line argument:
+
+```shell
+modelardb 127.0.0.1 path_to_file_with_sql_statements.sql
+```
+
+`modelardbd` can also be queried programmatically [from many different programming languages](https://arrow.apache.org/docs/index.html) using Apache Arrow Flight. The following Python example shows how to execute a simple SQL query against `modelardbd` and process the resulting stream of data points using [`pyarrow`](https://pypi.org/project/pyarrow/) and [`pandas`](https://pypi.org/project/pandas/). A PEP 249 compatible connector is also available for Python in the form of [PyModelarDB](https://github.com/ModelarData/PyModelarDB).
+
+```python
+from pyarrow import flight
+
+flight_client = flight.FlightClient("grpc://127.0.0.1:9999")
+ticket = flight.Ticket("SELECT * FROM wind_turbine LIMIT 10")
+flight_stream_reader = flight_client.do_get(ticket)
+
+for flight_stream_chunk in flight_stream_reader:
+    record_batch = flight_stream_chunk.data
+    pandas_data_frame = record_batch.to_pandas()
+    print(pandas_data_frame)
+```
+
+### Load Data
```

Comment:
Can we use "load" and "ingest" interchangeably or should we stick to using one of them?

Reply:
Replaced `load` with `ingest` everywhere except for `bulk load` as it is a well-known term.